### PR TITLE
Add missing plugins

### PR DESCRIPTION
**Pull request by thejuliekramer**
- Adds datagovtheme plugin to datagov-theme-production.ini.template
- Add config to use bootstrap 2

### CODE REVIEW

**Comment by akariv** [comment]
@thejuliekramer is this ready to merge?

File: .docker-compose.datagov-theme.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ services:
           install_standard_ckan_extension_github -r GSA/ckanext-geodatagov &&\
           install_standard_ckan_extension_github -r akariv/ckanext-datajson -b datagov &&\
           install_standard_ckan_extension_github -r akariv/USMetadata -e ckanext-usmetadata &&\
-          install_standard_ckan_extension_github -r GSA/ckanext-datagovtheme
+          install_standard_ckan_extension_github -r thejuliekramer/ckanext-datagovtheme
```
Comment (adborden) [inline .docker-compose.datagov-theme.yaml:31]: This should be the canonical GSA no? Maybe just tag this PR as dependent on https://github.com/GSA/datagov-ckan-multi/issues/204?

File: docker-compose/ckan-conf-templates/datagov-theme-production.ini.template [hunk 2]
```diff
@@ -83,11 +83,11 @@ ckan.plugins = stats text_view image_view recline_view datastore xloader
     structured_data geodatagov geodatagov_miscs z3950_harvester 
     arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection 
     geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_query 
-    datajson_harvest datajson
+    usmetadata datagovtheme datajson_harvest datajson spatial_metadata
```
Comment (avdata99) [inline docker-compose/ckan-conf-templates/datagov-theme-production.ini.template:86]: We do not need to activate the _usmetadata_ plugin here. It's just for the _inventory_ site. Here we work with the catalog. Am I ok @akariv?

Comment (akariv) [reply docker-compose/ckan-conf-templates/datagov-theme-production.ini.template:86]: Yes, that's true. 
Also note that while this file is used in development, it's not part of the production setup
(This is the one: https://github.com/ViderumGlobal/ckan-cloud-helm/blob/master/ckan/templates/ckan-configmap.yaml#L38)

File: docker-compose/ckan-conf-templates/datagov-theme-production.ini.template [hunk 3]
```diff
@@ -83,11 +83,11 @@ ckan.plugins = stats text_view image_view recline_view datastore xloader
...
   
 # Define which views should be created by default
 # (plugins must be loaded in ckan.plugins)
-ckan.views.default_views = image_view text_view recline_view
+# ckan.views.default_views = image_view text_view recline_view
```
Comment (akariv) [inline docker-compose/ckan-conf-templates/datagov-theme-production.ini.template:90]: Why are these being disabled?

**Comment by akariv** [comment]
@adborden is the data-gov theme needed in all instances being deployed on the multi-tenant cluster, or just the 'data.gov' ones?

**Pull request closed without merging**